src/components/SavedCardDetails/SavedCardDetails.css

```diff
@@ -0,0 +1,144 @@
+@import '../../marketplace.css';
+
+.savedPaymentMethodTitle {
+  @apply --marketplaceH4FontStyles;
+  font-weight: bold;
+  cursor: pointer;
+}
+
+.savedPaymentMethod {
+  display: inline-block;
+  width: 80%;
+}
+
+.cardIcon {
+  margin: 0 16px;
+}
+
+.closeIcon {
+  @apply --marketplaceModalCloseIcon;
+  margin-right: 5px;
+}
+
+.expirationDate {
+  padding-left: 24px;
+}
+
+.expirationDateExpired {
+  padding-left: 24px;
+  color: var(--failColor);
+}
+
+.cardExpiredText {
+  @apply --marketplaceH4FontStyles;
+  color: var(--failColor);
+  font-weight: var(--fontWeightSemiBold);
+}
+
+.menu {
+  width: 564px;
+  max-width: 100%;
+  height: auto;
+}
+
+.menuLabel {
+  text-align: left;
+  padding: 0;
+}
+
+.menuLabelWrapper {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  border: 1px solid var(--matterColorNegative);
+  border-radius: var(--borderRadius);
+  box-shadow: var(--boxShadow);
+  padding: 16px;
+}
+
+.menuLabelWrapperExpired {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  border: 1px solid var(--failColor);
+  border-radius: var(--borderRadius);
+  box-shadow: var(--boxShadow);
+  padding: 16px;
+}
+
+.menuContent {
+  border-radius: var(--borderRadius);
+  box-shadow: var(--boxShadow);
+  border: 1px solid var(--matterColorNegative);
+  padding: 16px 0;
+}
+
+.menuItem {
+  color: var(--matterColor);
+  &:hover {
+    background-color: var(--matterColorNegative);
+  }
+}
+
+.menuText {
+  padding: 16px 24px;
+  @apply --marketplaceMessageFontStyles;
```

Comment:
Try to introduce @apply property-set as early as possible, if it contains paddings it would overwrite previously set padding rule.